docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: "3.2"
```

Comment:
Does this Compose file use features that require this version change?

Reply:
No, I was just including the upstream change from the skeleton.  I will revert it.

Reply:
Please see commit ff04586.